_meta/mozilla-mobile-focus-android.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "mozilla-mobile/focus-android",
+  "revs": {
+    "master": "62e101a4caa9cac8be4bc71660217bce1c12fc43"
+  }
+}
```

Comment:
Flod mentioned on Slack he wasn't sure if this should be automation creating this - vs this PR?

Reply:
Since this PR already creates all the strings up to this commit, I want to signal to the automation that it does not need to go back further than this in the next sync.